graphene/types/base.py

```diff
@@ -1,6 +1,13 @@
+import six
+
 from ..utils.subclass_with_meta import SubclassWithMeta
 from ..utils.trim_docstring import trim_docstring
 
+if six.PY3:
+    from typing import Type
+else:
```

Comment:
since the typing in this file is just type hints, which are just comments to the python interpreter, do you even need this for py27? What happens if you just remove the `else` completely and so only have Type defined for py3?

Reply:
I gave it a shot in https://github.com/graphql-python/graphene/pull/913, seems like omitting the `else` statement tests still pass.

Reply:
Didn't think of that... checking

Reply:
@dan98765 ok closing this PR in favor of yours